src/app/learn/reference/faq/index.html

```diff
@@ -22,7 +22,7 @@
   </stache-page-anchor>
 
   <p>
-    In the <stache-code>package.json</stache-code> configuration file for NPM that the SKY UX template provides in the default SPA, update the <stache-code>@blackbaud/skyux</stache-code> property that specifies the version of SKY UX components. To find the latest version of the SKY UX components, check the <a href="https://github.com/blackbaud/skyux2/blob/master/CHANGELOG.md">changelog in the SKY UX repo</a> or the <a href="https://www.npmjs.com/package/@blackbaud/skyux">NPM page for the SKY UX package</a>.
+    In the <stache-code>package.json</stache-code> configuration file for NPM that the SKY UX template provides in the default SPA, update the <stache-code>@blackbaud/skyux</stache-code> property that specifies the version of SKY UX components. To find the latest version of the SKY UX components, check the <a routerLink="/contribute/packages">changelog in the component's repository</a> or the <a href="https://www.npmjs.com/package/@blackbaud/skyux">NPM page for the SKY UX package</a>.
```

Comment:
I think my notes for this section were wrong. When I audited the site, I thought that we just needed to update the paths for where to find components. But based on the section we added at https://developer.blackbaud.com/skyux/learn/get-started/basics/modules#update-sky-ux-modules, I don't think we want users manually updating versions in the `package.json` anymore. I think this FAQ item should reflect the content in that section instead. (Let me know if you want me to handle updating it.)